js/database.js

```diff
@@ -0,0 +1,66 @@
+import { getToday, incrementDay, isSameDate, MILISECOND_HOUR } from "./dateManipulation.js"
+
+if(!localStorage.getItem("eventId")){
+    localStorage.setItem("eventId", 0)
+}
+
+const findByDate = (date) => {
+    const nextEventId = localStorage.getItem("eventId")
```

Comment:
I would suggest calculating nextEventId instead of storing it in localStorage. You could have a database method for that. Let's say if you would be storing forms under key "forms", this method could potentially look something like: `Math.max(...Object.keys(JSON.parse(localStorage.getItem('form') || ''))) + 1`. You can split this line ofc to make it more readable :)